internal/datacoord/meta.go

```diff
@@ -248,11 +274,16 @@ func (m *meta) SetState(segmentID UniqueID, targetState commonpb.SegmentState) e
 	}
 	// Update in-memory meta.
 	m.segments.SetState(segmentID, targetState)
+	log.Info("meta update: setting segment state - complete",
+		zap.Int64("segment ID", segmentID),
+		zap.Any("target state", targetState))
```

Comment:
NIT: targetState has String() method, we could use `zap.String()` instead of `zap.Any()`

Reply:
Done.